### PR TITLE
Preparation for Ukrainian i8n 

### PR DESCRIPTION
**Pull request by orta**
/cc @AMashoshyna

### CODE REVIEW

File: packages/typescriptlang-org/scripts/makeMarkdownOfTranslations.js [hunk 1]
```diff
@@ -69,17 +72,21 @@ const toMarkdown = files => {
...
     todo.forEach(f => {
       md.push(markdownLink(f))
     })
+
+    totalDone += done.length
+    totalTodo += todo.length
   })
 
+  md[0] = `For this language there are ${totalDone} translated files, with ${totalTodo} TODO.\n\n`
   return md.join("\n")
```
Comment (orta) [inline packages/typescriptlang-org/scripts/makeMarkdownOfTranslations.js:90]: @khaosdoctor  - this should make your life easier with those numbers

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -41,3 +41,10 @@ packages/tsconfig-reference/copy/id/**/*.md @jefrydco @nusendra @mazipan @mandap
 packages/typescriptlang-org/src/copy/id/**/*.ts @jefrydco @nusendra @mazipan @mandaputtra [translation] [id]
 packages/typescriptlang-org/src/copy/id.ts @jefrydco @nusendra @mazipan @mandaputtra [translation] [id]
 packages/documentation/copy/id/**/*.md @jefrydco @nusendra @mazipan @mandaputtra [translation] [id]
+
+# Collaborators for Ukrainian Translation of the Website
+packages/playground-examples/copy/id @AMashoshyna [translation] [ua]
```
Comment (AMashoshyna) [inline .github/CODEOWNERS]:
```suggestion
packages/playground-examples/copy/id @AMashoshyna [translation] [uk]
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -41,3 +41,10 @@ packages/tsconfig-reference/copy/id/**/*.md @jefrydco @nusendra @mazipan @mandap
...
+packages/tsconfig-reference/copy/id/**/*.md @AMashoshyna [translation] [ua]
```
Comment (AMashoshyna) [inline .github/CODEOWNERS]:
```suggestion
packages/tsconfig-reference/copy/id/**/*.md @AMashoshyna [translation] [uk]
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -41,3 +41,10 @@ packages/tsconfig-reference/copy/id/**/*.md @jefrydco @nusendra @mazipan @mandap
...
+packages/typescriptlang-org/src/copy/id/**/*.ts @AMashoshyna [translation] [ua]
```
Comment (AMashoshyna) [inline .github/CODEOWNERS]:
```suggestion
packages/typescriptlang-org/src/copy/id/**/*.ts @AMashoshyna [translation] [uk]
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -41,3 +41,10 @@ packages/tsconfig-reference/copy/id/**/*.md @jefrydco @nusendra @mazipan @mandap
...
+packages/typescriptlang-org/src/copy/id.ts @AMashoshyna [translation] [ua]
```
Comment (AMashoshyna) [inline .github/CODEOWNERS]:
```suggestion
packages/typescriptlang-org/src/copy/id.ts @AMashoshyna [translation] [uk]
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -41,3 +41,10 @@ packages/tsconfig-reference/copy/id/**/*.md @jefrydco @nusendra @mazipan @mandap
...
+packages/documentation/copy/id/**/*.md @AMashoshyna [translation] [ua]
```
Comment (AMashoshyna) [inline .github/CODEOWNERS]:
```suggestion
packages/documentation/copy/id/**/*.md @AMashoshyna [translation] [uk]
```

File: packages/typescriptlang-org/scripts/updateGitHubTranslationIssues.ts [hunk 7]
```diff
@@ -13,6 +13,7 @@ const languages = {
   zh: 296,
   ko: 910,
   id: 938,
+  ua: 1149,
```
Comment (AMashoshyna) [inline packages/typescriptlang-org/scripts/updateGitHubTranslationIssues.ts]:
```suggestion
  uk: 1149,
```

**Review by AMashoshyna — COMMENTED** [review]
Would you mind to change `ua` to `uk`, as per ISO 639-1? UA stands for country.

**Comment by orta** [comment]
Yeah, great point!

**Pull request merged**